actionpack/lib/action_dispatch/http/mime_types.rb

```diff
@@ -21,7 +21,7 @@
 Mime::Type.register "application/xml", :xml, %w( text/xml application/x-xml )
```

Comment:
should `.xml` extension likewise be registered?

also, perhaps refactor `Mime::Type.register` API to use kwargs?